docs/user-guide/projects/project-lists/README.md

```diff
@@ -148,6 +155,46 @@ By selecting *Overall activity* you can open a view in which all the latest glob
 
 ![overall activity meeting filter](actvity-global-filter.png)
 
+## Favorite project lists
+
+You can mark your private project lists as favorites. This includes shared, public and private project lists, but not the static ones, e.g. *Active projects* or *Archived projects*. To mark a project list as favorite click the star icon in the top right corner. 
```

Comment:
@MayaBerd I would reformulate this as something like "You can mark project lists as favourites, both shared and private, but not the static ones..."